blockchain/blockchain-wallet-v4-frontend
# Feat/bsv get started

## Description
2044

## Change Type
- Feature

## Testing Steps
1. have bsv in wallet
2. open wallet and wait for modal
3. reload wallet, modal should not show

## Code Checklist
